src/DiscreteBayesNet/structure_scoring.jl

```diff
@@ -0,0 +1,18 @@
+"""
+    bayesian_score(g::DAG, names::Vector{Symbol}, data::DataFrame, prior::DirichletPrior=UniformPrior())
+
+Compute the bayesian score for graph structure `g`, with the data in `data`. `names` containes a symbol corresponding to each vertex in `g` that is the name of a column in `data`.
+
+Note that every entry in data must be an integer greater than 0
```

Comment:
@rejuvyesh That is simply how the Categorical distribution in Distributions.jl works. The domain is 1:n, not 0:n-1.